.github/workflows/publish.yml

```diff
@@ -1,12 +1,17 @@
 name: Publish to NPM
 on:
-  pull_request:
-    types: [closed]
-    branches:
-      - master
+  workflow_dispatch:
```

Comment:
Yep! If I interpret the [iOS](https://github.com/rive-app/rive-ios/blob/main/.github/workflows/release.yml#L4) and [Android](https://github.com/rive-app/rive-android/blob/master/.github/workflows/release.yml#L3) side, it's the same deal there. I think one potential ideal state that could be cool is if we could get something like conventional commits to dictate an automated release (i.e `"fix: rev cpp dependency"` automates a patch release; `"feat: add API for resizing canvas"` automates a minor release; `"breaking: Default to playing a state machine instead of an animation"` automates a major release) @tjfamodu might have thoughts around this